PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchJobRunner.java

```diff
@@ -137,15 +140,23 @@ public ISparkBatchJob buildSparkBatchJob(@NotNull SparkSubmitModel submitModel,
                 destinationRootPath = rawRootPath.endsWith("/") ? rawRootPath : rawRootPath + "/";
                 // e.g. for adl://john.azuredatalakestore.net/root/path, adlsAccountName is john
                 String adlsAccountName =  destinationRootPath.split("\\.")[0].split("//")[1];
-                SubscriptionDetail subscriptionDetail =
-                        AzureSparkClusterManager.getInstance().getSubscriptionDetailByStoreAccountName(adlsAccountName)
-                                .toBlocking().singleOrDefault(null);
+               Optional<SubscriptionDetail> subscriptionDetail =  null;
```

Comment:
Should not assign `Optional` type to null.

```suggestion
               Optional<SubscriptionDetail> subscriptionDetail =  Optional.empty();
```